### PR TITLE
Update awsRegionUrls.json with new regions

### PR DESCRIPTION
**Pull request by triptec**
Used https://docs.aws.amazon.com/general/latest/gr/rande.html for reference

### CODE REVIEW

**Comment by triptec** [comment]
should close #72

File: lib/resources/awsRegionUrls.json [hunk 1]
```diff
@@ -3,15 +3,20 @@
   "us-east-1": "s3-website-us-east-1.amazonaws.com",
   "us-west-1": "s3-website-us-west-1.amazonaws.com",
   "us-west-2": "s3-website-us-west-2.amazonaws.com",
-  "ca-central-1": "s3-website.ca-central-1.amazonaws.com",
+  "af-south-1": "s3-website-af-south-1.amazonaws.com",
+  "ap-east-1": "s3-website-ap-east-1.amazonaws.com",
```
Comment (fernando-mc) [inline lib/resources/awsRegionUrls.json:7]: should this be - `s3-website.ap-east-1.amazonaws.com`
https://docs.aws.amazon.com/general/latest/gr/s3.html

File: lib/resources/awsRegionUrls.json [hunk 2]
```diff
@@ -3,15 +3,20 @@
...
   "ap-southeast-1": "s3-website-ap-southeast-1.amazonaws.com",
   "ap-southeast-2": "s3-website-ap-southeast-2.amazonaws.com",
   "ap-northeast-1": "s3-website-ap-northeast-1.amazonaws.com",
+  "ca-central-1": "s3-website.ca-central-1.amazonaws.com",
   "eu-central-1": "s3-website.eu-central-1.amazonaws.com",
   "eu-west-1": "s3-website-eu-west-1.amazonaws.com",
   "eu-west-2": "s3-website.eu-west-2.amazonaws.com",
   "eu-west-3": "s3-website.eu-west-3.amazonaws.com",
+  "eu-north-1": "s3-website.eu-north-1.amazonaws.com",
+  "me-south-1": "s3-website.me-south-1.amazonaws.com",
```
Comment (fernando-mc) [inline lib/resources/awsRegionUrls.json:20]: I don't see this one listed?

File: lib/resources/awsRegionUrls.json [hunk 3]
```diff
@@ -3,15 +3,20 @@
   "us-east-1": "s3-website-us-east-1.amazonaws.com",
   "us-west-1": "s3-website-us-west-1.amazonaws.com",
   "us-west-2": "s3-website-us-west-2.amazonaws.com",
-  "ca-central-1": "s3-website.ca-central-1.amazonaws.com",
+  "af-south-1": "s3-website-af-south-1.amazonaws.com",
```
Comment (fernando-mc) [inline lib/resources/awsRegionUrls.json:6]: I don't see this one listed?
https://docs.aws.amazon.com/general/latest/gr/s3.html

**Comment by triptec** [comment]
Oh, I was obviously making assumptions I should not have. Updated the PR to just add what I know.

**Comment by fernando-mc** [comment]
Just cut a new release. This should be included in v2.6.0 @triptec. Thanks for the contribution!

**Pull request merged**